app/Http/Controllers/CategoriesController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\ValidateCategory;
+use App\Models\Categories;
+use Illuminate\Http\Request;
+use Illuminate\Support\Str;
+
+class CategoriesController extends Controller
+{
+    public function index()
+    {
+        $categories = Categories::all();
```

Comment:
Quando for usar esses metodos estáticos da model de operações, usa da seguinte forma:
```suggestion
        $categories = Categories::query()->all();
```
Desse jeito o teu editor tem como fazer autocomplete dos metodos disponiveis, se tu digitar ` Categories::query()->` vai ver que varios metodos aparecem no autocomplete e tu consegue acessar a documentação dos métodos (parametros que recebe, tipo do retorno etc...).

Da pra ti refatorar e mudar nos outros exemplos que tu tem no resto do código.

Isso é uma coisa que poucos sabem/usam mas é bem útil